scripts/iscan

```diff
@@ -257,12 +265,12 @@ cmd_scan() {
             --mount readonly,type=bind,source="$volume",destination=/vol \
             --env STEM="$name" \
             $env_bitdefender \
-            $image --for $opt_for /vol
+            $image --for $opt_for "${params[@]}" /vol
     elif [[ -b "$volume" ]]; then
         $DOCKER run --rm --privileged \
             --env STEM="$name" \
             $env_bitdefender \
-            $image --for $opt_for "$volume"
+            $image --for $opt_for "${params[@]}" "$volume"
```

Comment:
It will be simpler to just pass `ISCAN_EXTRA_PARAMS` environment variable through:
```sh
$DOCKER run ... ${ISCAN_EXTRA_PARAMS:+-e ISCAN_EXTRA_PARAMS="$ISCAN_EXTRA_PARAMS"} ...
```

Reply:
done